tests/main.js

```diff
@@ -468,30 +496,11 @@ test('[bundler: esbuild] Include most files from node modules present in `extern
   t.true(htmlExists)
 })
 
-test('[bundler: legacy] Includes specific Next.js dependencies when using next-on-netlify', async (t) => {
-  const { tmpDir } = await zipNode(t, 'node-module-next-on-netlify', { bundler: 'legacy' })
-  const [constantsExists, semverExists, otherExists, indexExists] = await Promise.all([
-    pathExists(`${tmpDir}/src/node_modules/next/dist/next-server/lib/constants.js`),
-    pathExists(`${tmpDir}/src/node_modules/next/dist/compiled/semver.js`),
-    pathExists(`${tmpDir}/src/node_modules/next/dist/other.js`),
-    pathExists(`${tmpDir}/src/node_modules/next/index.js`),
-  ])
-  t.true(constantsExists)
-  t.true(semverExists)
-  t.false(otherExists)
-  t.false(indexExists)
-})
+test('[bundler: esbuild] Does not throw if one of the modules defined in `externalModules` does not exist', async (t) => {
```

Comment:
I expect this to throw since:
1. External - don't bundle but  include in zip (so if missing fail).
2. Ignored - don't bundle and don't include in zip (so don't fail if missing).

Does that make sense?

Reply:
Ok - so per an internal discussion let's not throw on `external` until we expose them as a configuration.